sentry/logutils.py

```diff
@@ -1,104 +1,12 @@
 # Copyright 2016-2017 Versada <https://versada.eu/>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-import logging
-import urllib.parse
-
-import odoo.http
-
-_logger = logging.getLogger(__name__)
-try:
-    from raven.handlers.logging import SentryHandler
-    from raven.processors import SanitizePasswordsProcessor
-    from raven.utils.wsgi import get_environ, get_headers
-except ImportError:
-    _logger.debug('Cannot import "raven". Please make sure it is installed.')
-    SentryHandler = object
-    SanitizePasswordsProcessor = object
-
-
-def get_request_info(request):
-    '''
-    Returns context data extracted from :param:`request`.
-
-    Heavily based on flask integration for Sentry: https://git.io/vP4i9.
-    '''
-    urlparts = urllib.parse.urlsplit(request.url)
-    return {
-        'url': '%s://%s%s' % (urlparts.scheme, urlparts.netloc, urlparts.path),
-        'query_string': urlparts.query,
-        'method': request.method,
-        'headers': dict(get_headers(request.environ)),
-        'env': dict(get_environ(request.environ)),
-    }
-
-
-def get_extra_context():
-    '''
-    Extracts additional context from the current request (if such is set).
-    '''
-    request = odoo.http.request
-    try:
-        session = getattr(request, 'session', {})
-    except RuntimeError:
-        ctx = {}
-    else:
-        ctx = {
-            'tags': {
-                'database': session.get('db', None),
-            },
-            'user': {
-                'login': session.get('login', None),
```

Comment:
@fernandahf 
Could you check how to add the login and uid using new package, please?

I mean,
 - ![Screen Shot 2020-11-30 at 21 23 16](https://user-images.githubusercontent.com/6644187/100693350-4b624b80-3352-11eb-9d8b-dedd67280e44.png)

Now, it is not added

Reply:
FYI I just found the documentation for this:
 - https://docs.sentry.io/platforms/python/guides/django/enriching-events/identify-user/

Reply:
Done

Reply:
Done